code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -914,3 +914,25 @@
 		M.adjustToxLoss(-1, 0)
 	..()
 	. = TRUE
+
+/datum/reagent/consumable/lean
+	name = "Lean"
+	description = "The drank that makes you go wheezy."
+	color = "#DE55ED"
+	quality = DRINK_NICE
+	taste_description = "you can taste purple and a hint of opiod."
```

Comment:
Taste description doesn't work this way. The message gets constructed of all reagents that are contained.
```suggestion
	taste_description ="purple with a hint of opoid"
```